lib/datasource/maven/common.ts

```diff
@@ -1 +1,5 @@
 export const id = 'maven';
+
+export const MAVEN_REPO = 'https://repo.maven.apache.org/maven2';
+export const MAVEN_REPO_DEPRECATED = 'https://central.maven.org/maven2';
+export const CLOJARS_REPO = 'https://clojars.org/repo';
```

Comment:
`https://repo.clojars.org`? See https://clojars.org/.

Shouldn't this be moved to `deps-edn`, because it's only used there?

Reply:
Not only, it's also used with`leiningen`

Reply:
Could we simplify any logic by splitting the maven datasource into multiple?

Reply:
Don't think so, `sbt` already is separate, but clojure differs only with additional `registryUrl`